InnoClinic.OfficesAPI/BLL/Validators/OfficeForCreationDtoValidator.cs

```diff
@@ -0,0 +1,42 @@
+﻿using BLL.DTO;
+using BLL.Helpers;
+using FluentValidation;
+
+namespace BLL.Validators;
+
+public class OfficeForCreationDtoValidator : AbstractValidator<OfficeForCreatingDto>
```

Comment:
Too many duplications in `OfficeForCreationDtoValidator` and `OfficeForUpdatingDtoValidator`. Can we do anything to avoid them?